heron/common/src/python/utils/__init__.py

```diff
@@ -134,7 +132,7 @@ def get_heron_dir():
   This will extract heron directory from .pex file.
   :return: root location for heron-cli.
   """
-  path = "/".join(os.path.realpath(__file__).split('/')[:-7])
+  path = "/".join(os.path.realpath(__file__).split('/')[:-8])
```

Comment:
Could you include an example of an expected file format here, to illustrate the parsing logic.